### PR TITLE
Seed Turbo Fuse pool with Fei

### PR DESCRIPTION
**Pull request by thomas-waite**
# Summary
Seeds the Turbo Fuse pool with $10M of Fei. Liquidity provided by the OA multisig timelock, which currently has ~$50M.

### CODE REVIEW

File: proposals/dao/fip_97.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  return {
+    turboFusePCVDeposit
+  };
+};
+
+// Do any setup necessary for running the test.
+// This could include setting up Hardhat to impersonate accounts,
+// ensuring contracts have a specific state, etc.
+const setup: SetupUpgradeFunc = async (addresses, oldContracts, contracts, logging) => {
+  console.log(`No actions to complete in setup for fip${fipNumber}`);
```
Comment (Joeysantoro) [inline proposals/dao/fip_97.ts:43]: You'll need to impersonate and whitelist the PCV deposit to deposit to the turbo pool

File: protocol-configuration/mainnetAddresses.ts [hunk 2]
```diff
@@ -1855,6 +1855,11 @@ const MainnetAddresses: MainnetAddresses = {
     artifactName: 'RariGovernanceTokenUniswapDistributor',
     address: '0x1fa69a416bcf8572577d3949b742fbb0a9cd98c7',
     category: AddressCategory.Governance
+  },
+  rariTurboFusePool: {
+    artifactName: 'unknown',
+    address: '0x1d9EEE473CC1B3b6D316740F5677Ef36E8f0329e',
```
Comment (Joeysantoro) [inline protocol-configuration/mainnetAddresses.ts:1861]: This should be the cToken I believe: [0x081E7C60bCB8A2e7E43076a2988068c0a6e69e27](https://etherscan.io/address/0x081E7C60bCB8A2e7E43076a2988068c0a6e69e27)

Comment (thomas-waite) [reply protocol-configuration/mainnetAddresses.ts:1861]: You're right, was hunting around for that. Thanks

File: proposals/dao/fip_97.ts (hunk 1 above)
Comment (thomas-waite) [reply proposals/dao/fip_97.ts]: Done. Note to self - whitelist this PCV deposit on the TurboAdmin once it has been deployed

File: proposals/description/fip_97.ts [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      values: '0',
+      method: 'deposit()',
+      arguments: [],
+      description: 'Deposit ~$10M of Fei into Turbo Fuse Pool'
+    },
+    {
+      target: 'collateralizationOracle',
+      values: '0',
+      method: 'addDeposits(address[])',
+      arguments: [['{turboFusePCVDeposit}']],
```
Comment (Joeysantoro) [inline proposals/description/fip_97.ts:24]: Ah. You'll need to do a wrapper because the PCV deposit doesn't accurately report FEI

**Review by Joeysantoro — DISMISSED** [review]
Approved assuming tests pass

**Review by Joeysantoro — APPROVED** [review]

**Pull request merged**